console-subscriber/src/aggregator.rs

```diff
@@ -498,3 +531,10 @@ impl Task {
         }
     }
 }
+
+fn serialize_histogram(histogram: &Histogram<u64>) -> Result<Vec<u8>, V2DeflateSerializeError> {
+    let mut serializer = V2DeflateSerializer::new();
```

Comment:
what's the motivation for the V2 + DEFLATE serializer, versus the V2 without compression? I'm assuming the intention is to reduce the wire size of the histogram. looking at the [`hdrhistogram` docs, it seems like adding the compression has a significant performance penalty](https://docs.rs/hdrhistogram/7.3.0/hdrhistogram/serialization/index.html#performance-concerns) --- they seem to mostly recommend using compression for archival storage.

not necessarily saying this is the wrong choice, but it seems like it would be good to make sure we have a clear understanding of the performance tradeoffs, before we commit to having a particular serialization format be part of the API. of course, we can break the protobuf format as much as we like, currently, because we haven't committed to a stable release yet...but this will be harder in the future.

Reply:
I just went with the smallest size option initially. Thank you for pointing this out, the serializer format was definitely something I wanted to discuss. After re-reading the paragraph you linked I definitely agree. Also compression is something that can be applied at a lower layer.

So I'm going to change this to V2Serializer.